internal/services/openai.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 type OpenAI interface {
-	GetErrorCodesDescription(make, model string, year int32, errorCodes []string) (string, error)
+	GetErrorCodesDescription(make, model string, errorCodes []string) (string, error)
```

Comment:
I don't feel super strongly about this, but if you look at [the other services](https://github.com/DIMO-Network/devices-api/blob/9dc5829651b1000b1c112881df84b5b0a53562cb/internal/services/autopi_api_service.go#L25) they put a `//go:generate` comment at the top. Then `go generate ./...` builds all the mocks.